lit/docs/resources/managing.lit

```diff
@@ -28,3 +28,42 @@
   \italic{bosh-io-release} resource might use \code{version:11.2} in place of
   \code{ref:abcdef}.
 }
+
+\section{
+  \title{\code{fly pin-resource}}{fly-pin-resource}
+
+  To pin a resource to a specific version of that resource, run:
+
+  \codeblock{bash}{{{
+  $ fly -t example pin-resource --resource my-pipeline/my-resource \
+      --version ref:bceaf
+  }}}
+
+  Note that the version needs to be provided as key-value pairs. For the git
+  resource the \code{ref:} prefix is used. For example, the \italic{registry} 
+  resource might use \code{digest} as a prefix like \code{digest:sha256:94be7d7b}.
```

Comment:
I think there might be a simpler way to say this (because it's a really good point), but I'm ok with it as is.

Reply:
I know :thinking: I tried to re-formulate it a little but didn't have a better idea either. I could copy the sentence from line 27 or leave it out as well, but for me personally the syntax of versions was not very clear from that. So I tried to explain it here a little differntly.

Reply:
don't sweat it, it's already an improvement 😄. I might take a crack at rephrasing in a follow-on PR.